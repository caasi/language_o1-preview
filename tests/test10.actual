Error: Unknown AST node type '7'
