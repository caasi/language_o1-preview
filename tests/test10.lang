let infinite_recursion = \n. infinite_recursion n
infinite_recursion 1