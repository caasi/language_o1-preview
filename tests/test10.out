Error: Stack overflow due to infinite recursion
